### ISSUE TITLE
Feature request: new placeholders

### ISSUE THREAD

**Issue by EkonC** [CraftingStore/MinecraftPlugin#10]
Hello, I want to have a hall for my top donors with NPCs with their skins but there are no suitable placeholders for me.

Current top donator placeholders only show this format:
`MinecraftName: Amount spent this month`
I'd like to have placeholders for `MinecraftName` and `Amount spent this month` separately (to format them, use them as NPC names, NPC skins, in holograms etc.) as well as top donator not only this month but top donator ever.

It would be a great funcionality that would really help me

**Comment by w41k3r26** [comment]
Waiting on this

**Comment by BluBoy3ch0** [comment]
> Hello, I want to have a hall for my top donors with NPCs with their skins but there are no suitable placeholders for me.
> 
> Current top donator placeholders only show this format:
> `MinecraftName: Amount spent this month`
> I'd like to have placeholders for `MinecraftName` and `Amount spent this month` separately (to format them, use them as NPC names, NPC skins, in holograms etc.) as well as top donator not only this month but top donator ever.
> 
> It would be a great funcionality that would really help me

you could make a plugin, and set the store to run a command, then when that command runs with your plugin you can do things like save the amount purchased and who purchased it.

**Comment by EkonC** [comment]
Yes but the current plugin already supports placeholders, this was just a suggestion for making them better somehow.
